praetorian_cli/handlers/workflow.py

```
@@ -0,0 +1,330 @@
+    # Apply filter if provided
+    if pattern:
+        all_workflows = [wf for wf in all_workflows if fnmatch.fnmatch(wf.cli_name, pattern)]
+        if not all_workflows:
+            click.echo(f"No workflows matching pattern '{pattern}'.")
+            return
+
+    click.echo("Available workflows:\n")
+    for wf in all_workflows:
+        click.echo(f"  {wf.cli_name}")
+        click.echo(f"    {wf.description}")
+    click.echo(f"\nUse 'praetorian chariot agent workflow info <name>' for parameter details.")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)